### ISSUE TITLE
Longer texts in table cells are truncated

### ISSUE THREAD

**Issue by gy-mate** [charmbracelet/mods#360]
**Describe the bug**
Longer texts in Markdown table cells of the response are truncated instead of line wrapping them.

**Setup**
Please complete the following information along with version numbers, if applicable.
 - mods `1.6.0`
 - macOS `15.0`
 - zsh
 - Terminal.app or Warp `v0.2024.09.17.08.02.stable_01`

**To Reproduce**
1. Run the following command:
  ```bash
  mods 'What is the difference between the features of Collection and Set? Give me a short comparison in a table.'
  ```
2. See the truncated texts in the table cells of the answer:
  <img width="837" alt="Screenshot 2024-09-25 at 10 35 09" src="https://github.com/user-attachments/assets/9fc62792-e5a2-4862-99f2-ee3f4dc4aa70">
  <img width="1043" alt="Screenshot 2024-09-25 at 10 38 44" src="https://github.com/user-attachments/assets/3415a897-0adf-4d53-8fae-1d873122c52e">

**Expected behavior**
Texts are line wrapped and fully visible.

**Additional context**
_This issue might belong to [ charmbracelet/glamour](https://github.com/charmbracelet/glamour)._

**Comment by caarlos0** [comment]
yeah its how glamour works at the moment. This might change in the future, but no idea when.

**Comment by gy-mate** [comment]
@caarlos0 Could you please transfer this issue to https://github.com/charmbracelet/glamour then (if it belongs there)? Thanks a lot in advance! :)

**Comment by andreynering** [comment]
Hi @gy-mate,

We made several improvements on table rendering in Glamour in the past months, and we are already using the newer version on `main`, it was just not released in `mods` yet.

You can build from `main` for now.

<details><summary>Release</summary>
<p>

<img width="794" alt="Image" src="https://github.com/user-attachments/assets/a9280d72-9c84-4e0c-bfe7-26bdee23863d" />

</p>
</details> 

<details><summary>main</summary>
<p>

<img width="766" alt="Image" src="https://github.com/user-attachments/assets/0a48f101-8398-428d-bd08-8ae593394004" />

</p>
</details>

**Comment by gy-mate** [comment]
Awesome, thank you! :)
